io.openems.common/src/io/openems/common/utils/FileUtils.java

```diff
@@ -0,0 +1,34 @@
+package io.openems.common.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class FileUtils {
+
+    private static final Logger log = LoggerFactory.getLogger(FileUtils.class);
+
+    public static StringBuilder checkAndGetFileContent(String path) {
```

Comment:
I like the idea of introducing a FileUtils class. In my opinion `checkAndGetFileContent` should throw an (OpenEMS)Exception if it fails, otherwise it is hard to handle the error case in the calling method.

In my experience at FENECON, once you have multiple OpenEMS Edge instances setup, nobody is going to actually read the logs and detect errors like that. It's crucial to always throw Exceptions and handle them in a way, they finally become visible - e.g. using StateChannels; see the "RunFailed"-Controller-Channel as an example: https://github.com/OpenEMS/openems/blob/develop/io.openems.edge.core/src/io/openems/edge/core/cycle/CycleWorker.java#L101. The StateChannel "RunFailed" is defined as a "Fault"-Channel and is set when a Controller run()-method failes. This causes eventually the "_sum/State" to be "Fault", which makes it detectable for a central Dashboard in our office.